nin/frontend/app/scripts/directives/waveform.js

```diff
@@ -1,19 +1,54 @@
 const Waveform = require('../../lib/waveform');
 
 function waveform($window) {
+
```

Comment:
Ref style discussion, i prefer no newline after function declarations.
How about the rest of you?